db/migrate/20220711070430_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :email
+      t.string :first_name
+      t.string :last_name
+      t.string :role
+      t.string :password
```

Comment:
```suggestion
      t.string : password_digest
```

https://hackernoon.com/authentication-and-authorization-with-bcrypt-in-rails-mw1g3u3l#:~:text=must%20have%20a-,password_digest,-attribute.%20In%20your